DirectusExtensions/charging-stations-bundle/src/endpoints-charging-stations/index.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright Contributors to the CitrineOS Project
+//
+// SPDX-License-Identifier: Apache 2.0
+
+import { defineEndpoint } from '@directus/extensions-sdk';
+
+export default defineEndpoint({
+    id: 'charging-stations',
+    handler: (router, { database }) => {
+        router.post('/update-station-status', async (req, res) => {
```

Comment:
@thanaParis why dont you just use one of the existing endpoints to update the record? Are you sure this needs to be a custom endpoint?

Reply:
There is no existing endpoint that accepts the output of the subscription api.

Reply:
You can use `/items/ChargingStations` endpoints that are available in Directus by default to trigger:

```
await database('ChargingStations')
                    .where({ id: stationId })
                    .update({ isOnline });
```

so what is the benefit of having this as a separate endpoint?

Reply:
You can review the structure of the subscription api in OcppRouter/router.ts. They don't match, unfortunately, and shouldn't since they serve a wider purpose than just providing Directus with updates.